provisions/group_vars/all

```diff
@@ -20,7 +20,6 @@ db_pass: cccp
 db_host: '0.0.0.0'
 db_port: 5432
 allowed_hosts: "['127.0.0.1']"
-db_backup_host_path: "/srv/pipeline-logs/db/cccp/"
```

Comment:
I believe this can fails if we dont have this value. The CI inventory has specified the value, does not mean its there in any other inventory file, unless we want to make it compulsory in inventory file.